packages/core/src/navigator/ActiveScreenProvider.ts

```diff
@@ -112,3 +125,55 @@ export const useContextRegister = (data: any) => {
     }
   }, [data, isFocused]);
 };
+
+export const isModel = (
+  context: Data,
+  model: string,
+  key?: string,
+): boolean => {
+  if (!Array.isArray(context.models) || context.models.length === 0) {
+    return false;
+  }
+
+  return context.models.some(m => {
+    const isModelNameMatch = m.model === model;
+
+    if (key) {
+      return isModelNameMatch && m.key === key;
+    }
+
+    return isModelNameMatch;
```

Comment:
Create helper to manage this filter as you use it here to check if it exists and below to fetch model